src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -182,49 +181,55 @@ export const PropertyList = ({
         dispatch({ type: SET_OAT_PROPERTY_EDITOR_MODEL, payload: newModel });
     };
 
+    const handleSelectorPosition = (e) => {
+        const boundingRect = e.target.getBoundingClientRect();
+        setPropertySelectorPosition({
+            ...propertySelectorPosition,
+            top: boundingRect.top,
+            left: boundingRect.left
+        });
+        setPropertySelectorTriggerElementsBoundingBox(boundingRect);
+    };
+
+    const getIsMouseLeaveNotNorth = (e) => {
```

Comment:
This can be simplified... i.e. `return e.clientY >= propertySelectorTriggerElementsBoundingBox.bottom ||
            e.clientX < propertySelectorTriggerElementsBoundingBox.left ||
            e.clientX > propertySelectorTriggerElementsBoundingBox.right;`

Reply:
Also... what exactly is this function doing? I'm not sure if the name is helping 🤔